.gitlab-ci.yml

```diff
@@ -1,242 +1,4 @@
-  <<: *setup_bundler_env
-  stage: 'sanity'
-  tags: ['docker']
-  script:
-    - 'if `hash apt-get`; then apt-get update; fi'
-    - 'if `hash apt-get`; then apt-get install -y rpm; fi'
-    - 'bundle exec rake check:dot_underscore'
-    - 'bundle exec rake check:test_file'
-    - 'bundle exec rake pkg:check_version'
-    - 'bundle exec rake pkg:compare_latest_tag'
-    - 'bundle exec rake pkg:create_tag_changelog'
-    - 'bundle exec puppet module build'
-
-# Linting
-#-----------------------------------------------------------------------
-
-pup5-lint:
-  <<: *pup_5
-  <<: *lint_tests
-
-pup6-lint:
-  <<: *pup_6
-  <<: *lint_tests
-
-# Unit Tests
-#-----------------------------------------------------------------------
-
-pup5-unit:
-  <<: *pup_5
-  <<: *unit_tests
-
-pup5.5.7-unit:
-  <<: *pup_5_5_7
-  <<: *unit_tests
-
-pup6-unit:
-  <<: *pup_6
-  <<: *unit_tests
-
-# Acceptance tests
-# ==============================================================================
-pup5.5.7:
-  <<: *pup_5_5_7
-  <<: *acceptance_base
-  script:
-    - 'bundle exec rake beaker:suites[default]'
-
-pup5.5.7-fips:
-  <<: *pup_5_5_7
-  <<: *acceptance_base
-  script:
-    - 'BEAKER_fips=yes bundle exec rake beaker:suites[default]'
-
-pup5.5.7-oel:
-  <<: *pup_5_5_7
-  <<: *acceptance_base
-  script:
-    - 'bundle exec rake beaker:suites[default,oel]'
-
-pup5.5.7-oel-fips:
-  <<: *pup_5_5_7
-  <<: *acceptance_base
-  <<: *only_with_SIMP_FULL_MATRIX
-  script:
-    - 'BEAKER_fips=yes bundle exec rake beaker:suites[default,oel]'
-
-pup5.5.7-security_modules:
-  <<: *pup_5_5_7
-  <<: *acceptance_base
-  script:
-    - 'bundle exec rake beaker:suites[security_modules]'
-
-pup5.5.7-security_modules-fips:
-  <<: *pup_5_5_7
-  <<: *acceptance_base
-  script:
-    - 'BEAKER_fips=yes bundle exec rake beaker:suites[security_modules]'
-
-pup5.5.7-oel:
-  <<: *pup_5_5_7
-  <<: *acceptance_base
-  script:
-    - 'bundle exec rake beaker:suites[default,oel]'
-
-pup5.5.7-oel-fips:
-  <<: *pup_5_5_7
-  <<: *acceptance_base
-  <<: *only_with_SIMP_FULL_MATRIX
-  script:
-    - 'BEAKER_fips=yes bundle exec rake beaker:suites[default,oel]'
-
-pup5.5.7-oel-security_modules:
-  <<: *pup_5_5_7
-  <<: *acceptance_base
-  script:
-    - 'bundle exec rake beaker:suites[security_modules,oel]'
-
-pup5.5.7-oel-security_modules-fips:
-  <<: *pup_5_5_7
-  <<: *acceptance_base
-  <<: *only_with_SIMP_FULL_MATRIX
-  script:
-    - 'BEAKER_fips=yes bundle exec rake beaker:suites[security_modules,oel]'
-
-pup5.5.7-compliance-fips:
-  <<: *pup_5_5_7
-  <<: *compliance_base
-  script:
-    - 'BEAKER_fips=yes bundle exec rake beaker:suites[compliance]'
-
-pup6:
-  <<: *pup_6
-  <<: *acceptance_base
-  script:
-    - 'bundle exec rake beaker:suites[default]'
-
-pup6-compliance-fips:
-  <<: *pup_6
-  <<: *acceptance_base
-  script:
-    - 'BEAKER_fips=yes bundle exec rake beaker:suites[compliance]'
+include:
+  - project: 'simp/remote-gitlab-ci'
+    file:    '/gitlab-templates/.basic.gitlab-ci.yml'
```

Comment:
(Following up on today's discussion:) Do not merge this yet.

This particular use of `include` and the current architecture of [simp/remote-gitlab-ci](https://github.com/simp/remote-gitlab-ci) will cause increasingly problematic RELENG-related issues over time.  This is a design issue, and it must be addressed _before_ introducing a centralized `include` strategy across all our repositories, or we may irretrievably lose the ability to run meaningful pipelines across different projects over time.

I'll follow up in more detail in issue [SIMP-5650](https://simp-project.atlassian.net/browse/SIMP-5650).